instrumentation/spring-amqp/README.md

```diff
@@ -0,0 +1,23 @@
+# brave-instrumentation-spring-amqp
+
+## Tracing for Spring Rabbit
+This module provides instrumentation for spring-amqp based services. Currently, the only broker supported by spring-amqp is RabbitMQ.
+
+## Configuration
```

Comment:
Since this uses a non-default dependency on spring-aop, might want to mention this dependency is required (provided won't do this)